internal/dag/status_test.go

```diff
@@ -1534,6 +1534,12 @@ func TestDAGStatus(t *testing.T) {
 				Conditions: []contour_api_v1.MatchCondition{{
 					Prefix: "/blog",
 				}},
+			}, {
```

Comment:
It'd be good to keep the original test case around too, i.e. a "simple" inclusion of two different child proxies with duplicate conditions, where now the first one is still programmed, and the second one is marked orphaned/not programmed.